NEW_PROJECTS.md

```diff
@@ -0,0 +1,70 @@
+This is an evolving set of guidelines and instructions for how to bring your existing project, or idea for a completely fresh project, under the SPIFFE organization.
+
+If your project is already under the SPIFFE banner, you may be looking for [MATURITY](/MATURITY.md) instead.
+
+## Notice
+
+Maintainer groups are given significant freedom in managing their projects to that we may best foster growth and innovation. There are some elements of SPIFFE subproject ownership to be aware of:
+The [SPIFFE Steering Committee](/ssc/README.md) (SSC) governs the SPIFFE organization as a whole, and has the final authority on acceptance, rejection, or eviction of projects. Additionally, as stewards of community health, the SSC is responsible for intervening in any [Code of Conduct](/CODE-OF-CONDUCT.md) violations or derelictions of maintainer responsibility.
+Note that when contributing a project to SPIFFE, its purpose becomes to serve the SPIFFE community rather than any personal gain. Additionally, we expect a sufficient number of maintainers for a project as it grows, not all of whom may bring the same direction you may originally have had in mind.
+Carefully consider this change in control and purpose.
+
+*Every SPIFFE project* at minimum must follow the [SPIFFE Code of Conduct](/CODE-OF-CONDUCT.md).
+
+## Project Proposal
+
+### Contact
+
+The SSC can be contacted with proposal details at ssc@spiffe.io or on [our SPIFFE slack channel](https://spiffe.slack.com/archives/C01GQ267JJU).
+
+### Proposal Outline
+
+The following details are expected for proposals to help us best understand how the project will serve the community.
+
+Additionally, the information here make great initial `README.md`, `CONTRIBUTING.md`, and `GOVERNANCE.md` files for the repo!
+
+#### Initial Maintainer List
+
+The list of initial maintainers of the project, including GitHub handle.
+
+#### Project Purpose and Direction
+
+What need is the project solving for the community? What problem exists or what enhancement will this create?
+
+What is the *direction* of the project? This is similar but distinct from the *purpose* of the project.
+
+While the purpose highlights the "Why" and "What" of the project, the direction should be about the "How".
+This could include:
+- What does "done" mean for the project? What is the idealized vision for the shape of the project? What are the milestones to complete for each level of [maturity](/MATURITY.md)?
+- What are practices or styles that will be encouraged or discouraged?
+- What notable features or options are in-scope or out-of-scope for the project? This especially will help differentiate your project from any similar ones that might already exist.
+
+It's ok if there's not fully fleshed-out answers to each of these, and projects evolve over time. However the more information that can be provided the more easily SSC may understand the proposal, and the more confidence there will be in alignment amongst the maintainers.
+
+#### Governance
```

Comment:
perhaps add a bit about conflict resolution mechanisms within the team of maintainers.

Reply:
Not sure… “Governance” is better I think.

Reply:
added